src/test/java/integration/ShipmentControllerIT.java

```diff
@@ -94,20 +96,23 @@ public void getShipment_notFound() throws Exception {
 
     @Test
     @SuppressWarnings("unchecked")
-    public void createShipment() throws Exception {
-        Client sender = testHelper.createClient();
+    public void createShipment_presavedClients_trasmittingUuidOnly() throws Exception {
```

Comment:
Plz, rename metho to
createpShipment_presavedSender_presavedRecipient
or
createpShipment_presavedSenderAndRecipient
in order to have one naming style in all your test methods.